src/vm/wren_core.c

```diff
@@ -942,7 +942,7 @@ DEF_PRIMITIVE(num_fromString)
   double number = strtod(string->value, &end);
 
   // Skip past any trailing whitespace.
-  while (*end != '\0' && isspace(*end)) end++;
+  while (*((unsigned char*)end) != '\0' && isspace(*((unsigned char*)end))) end++;
```

Comment:
:+1: It seems `isspace` in cygwin is a bit pickier than other C libs. @bjorn's suggestion is how I'd do this.